### ISSUE TITLE
Graylog 2.0.0 Web interface "Server currently unavailable"

### ISSUE THREAD

**Issue by ajpen** [Graylog2/graylog2-server#2760]
I'm receiving the following message from the web interface:
`We are experiencing problems connecting to the Graylog server running on https://172.25.10.129/api-browser/. Please verify that the server is healthy and working correctly.

Error message
cannot GET https://172.25.10.129/api-browser/system/cluster/node (404)
` 
When I try to reach the api browser I get the following screen: 

![chrome_2016-08-30_11-28-59](https://cloud.githubusercontent.com/assets/4834786/18095435/f8a2c8b4-6ea4-11e6-998d-fd7971ed1ea1.png)

## Expected Behavior

Access to the graylog web interface over HTTPS

## Current Behavior

Graylog web interface unable to reach graylog server. 

## Steps to Reproduce (for bugs)

1. Upgrade Graylog 1.x - Graylog 2.0.3
2. Reconfigure Nginx as a reverse proxy for HTTPS using [this guide](http://docs.graylog.org/en/2.0/pages/configuration/web_interface.html#nginx) 
3. Set Graylog to use 127.0.0.1:12599 for the web interface, 127.0.0.1:12900 for the rest api
4. reload Nginx, Graylog
5. Attempt to access Graylog Web Interface

## Your Environment

* Graylog Version: 2.0.3
* Elasticsearch Version: 2.3.5
* MongoDB Version: 3.2.3
* Operating System: RHEL 7.2
* Browser version: Google Chrome 52.0.2743.116

EDIT: Removed unnecessary details.

**Comment by ajpen** [comment]
Resolved. Issue was Nginx was not reloaded after an update on the configuration. Closing

**Comment by ajpen** [comment]
Reopened because Graylog web interface works but the api-browser looks like this:

![chrome_2016-08-30_12-13-11](https://cloud.githubusercontent.com/assets/4834786/18097222/2257dc7a-6eab-11e6-9df0-84aec753347a.png)

**Comment by ajpen** [comment]
I've investigated the contents of the files and found that all the script files contain:

```

<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="robots" content="noindex, nofollow">
    <meta charset="UTF-8">
    <title>Graylog Web Interface</title>
    <link rel="shortcut icon" href="/favicon.png">
    
  </head>
  <body>
    <script src="/config.js"></script>
    <script src="/vendor.js"></script>
    
    <script src="/polyfill.f2a88026f0d0e7332f17.js"></script>
    
    <script src="/plugin/org.graylog.plugins.pipelineprocessor.ProcessorPlugin/plugin.org.graylog.plugins.pipelineprocessor.PipelineProcessorPlugin.124facd49c87779f691b.js"></script>
    
    <script src="/plugin/org.graylog.plugins.map.MapWidgetPlugin/plugin.org.graylog.plugins.map.MapWidgetPlugin.3a6e86e960b80407f74e.js"></script>
    
    <script src="/plugin/org.graylog.plugins.enterprise_integration.EnterpriseIntegrationPlugin/plugin.org.graylog.plugins.enterprise_integration.EnterpriseIntegrationPlugin.7f1e11d52a66d206799c.js"></script>
    
    <script src="/plugin/org.graylog.plugins.collector.CollectorPlugin/plugin.org.graylog.plugins.collector.CollectorPlugin.74e8dfee2c1d6b11bc87.js"></script>
    
    <script src="/app.f2a88026f0d0e7332f17.js"></script>
    
  </body>
</html>
 
```

**Comment by joschi** [comment]
@ajpen Please post your complete Graylog and nginx configuration files.

It looks like you've got some of the listeners and URIs mixed up.

**Comment by ajpen** [comment]
Nginx: 

```
server {
        listen 80;
        return 301 https://$host$request_uri;
}

server {

        listen 443;

## 10.10.10.10 refers to the server's public IP
        server_name 10.10.10.10;

         # ssl configurations here

        access_log                      /var/log/nginx/graylog.access.log;


        # Redirects for graylog web interface
        location / {


                proxy_set_header        Host $host;
                proxy_set_header        X-Real-IP $remote_addr;
                proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header        X-Forwarded-Proto $scheme;

                proxy_set_header        X-Graylog-Server-URL https://10.10.10.10/api;
                proxy_http_version      1.1;

                proxy_pass              http://127.0.0.1:12599;

                proxy_read_timeout      90;
                proxy_redirect          http://127.0.0.1:12599 https://10.10.10.10;
        }

        location /api/ {

                proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header    Host $http_host;
                proxy_set_header        Host $host;
                proxy_set_header        X-Real-IP $remote_addr;


                proxy_pass          http://127.0.0.1:12900/;
        }
}
```

Graylog: 

```
rest_listen_uri = http://127.0.0.1:12900/
rest_enable_cors = true
content_packs_auto_load = grok-patterns.json
web_listen_uri = http://127.0.0.1:12599/
web_enable_cors = true
web_endpoint_uri = https://10.10.10.10:443/api/

```

The graylog conf is mostly comments so I've sent all the rest and web related parts.

**Comment by joschi** [comment]
@ajpen The API browser running on another context-path than `/` has been fixed in Graylog 2.1.0.

Please upgrade to Graylog 2.1.0-rc.1 or wait for imminent the GA release.

**Comment by ajpen** [comment]
got it. Thanks
